src/script/conversation/ConversationRepository.ts

```diff
@@ -1504,11 +1504,11 @@ export class ConversationRepository {
    * @param conversationEntity Conversation entity to delete
    * @param timestamp Optional timestamps for which messages to remove
    */
-  private _clear_conversation(conversationEntity: Conversation, timestamp?: number) {
+  private _clearConversation(conversationEntity: Conversation, timestamp?: number) {
```

Comment:
I guess not part of this PR.. but I think we stopped prepending `_` to private ones as it's clear now.

Reply:
Yeah I know. I will do that in another PR :)